dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/EmailActionlet.java

```diff
@@ -96,7 +95,7 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
         String condition = params.get("condition").getValue();
         String cc = params.get("cc").getValue();
         String bcc = params.get("bcc").getValue();
-
+        String customHeaders = params.get("customHeaders").getValue();
 
```

Comment:
It would be nice if we could re-use the CUSTOM_HEADERS const here

Reply:
I know this is legacy code, but we should probably check for the param and throw an IllegalArgumentException or  something to avoid NPE